test/runnable/istring.d

```diff
@@ -0,0 +1,72 @@
+/*
+PERMUTE_ARGS:
+REQUIRED_ARGS: -transition=interpolate
+*/
+import std.conv : text;
```

Comment:
We should try to avoid using Phobos in the DMD testsuite. Maybe we can add a very reduced and minimalistic version of `text`?